contract_show_invoice/views/contract_view.xml

```diff
@@ -3,26 +3,14 @@
     <data>
 
         <record id="act_analytic_invoices" model="ir.actions.act_window">
-            <field name="context">{'search_default_analytic_account_ids':
-                [active_id], 'default_analytic_account_ids': active_id}</field>
```

Comment:
@pedrobaeza this context is useless because analytic_account_ids is a computed field and the search is done by domain